### DISCUSSION TITLE
Do I need an OAuth token to test with personal access?

### DISCUSSION THREAD

**Discussion by jneb802** [etsy/open-api#1221] [Q&A]
I'm reading through the v3 API docs and see a detailed explanation on how to obtain a OAuth tokens (which also requires a callback URI). However, I'm still developing with the personal access and don't have a hosted server / website yet. 

Do I need to use OAuth tokens in order to submit api calls using personal access?

**Comment by turboturtleD** [comment]
Yes, you need to use OAuth(2) tokens for API calls.
